AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStorePrivate.h

```diff
@@ -79,6 +79,15 @@ static NSString *const kMSPendingOperationColumnName = @"pending_operation";
  */
 + (MSDBColumnsSchema *)columnsSchema;
 
+/**
+ * Return the table for a given partition.
```

Comment:
Is this the full partition or shortened one? I find it hard to keep track of which is being referred to in many places